### ISSUE TITLE
how to execute ?

### ISSUE THREAD

**Issue by tissatussa** [xu-shawn/Serendipity#77]
i successfully compiled your v0.2 source (PR 75) and after that i find these programs :

```
target/Serendipity-Test.jar
target/original-Serendipity-Test.jar
/Serendipity-Dev
```

all files are about 5.3 Mb in size.

why are the `.jar` files called "Test" ?
the 3rd file has postfix "Dev", i guess that's because i compiled the PR 75 version ?

which version should i use in CuteChess ?
if i use a `.jar` file, will `java -jar ./[name].jar` be the proper command ?

[ i'm on Xubuntu 22.04 ]

**Comment by xu-shawn** [comment]
TL;DR: Use `target/Serendipity-Test.jar` or `/Serendipity-Dev`

`/Serendipity-Dev` is a temporary fix for OpenBench, which does not well support Java at the moment. It can be executed as if it is a binary file, and it works on Linux and MacOS only.

`target/Serendipity-Test.jar` is the JAR file for Serendipity. It is the same as `/Serendipity-Dev` except that it must be ran like so: `java -jar target/Serendipity-Test.jar`

`target/original-Serendipity-Test.jar` is the JAR file of Serendipity without dependencies packaged. This file is likely not runnable due to missing dependencies, and using it is discouraged.
